src/components/KCollapse/KCollapse.vue

```diff
@@ -111,6 +121,25 @@ export default defineComponent({
         return ['leading', 'trailing'].includes(value)
       },
     },
+    type: {
+      type: String as PropType<'default' | 'tags'>,
+      required: false,
+      default: 'default',
+    },
+    tags: {
+      type: Array,
+      default: () => [],
+    },
+    tagsAppearance: {
+      type: String as PropType<'default' | 'light'>,
+      required: false,
+      default: 'default',
+    },
+    noCollapse: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
```

Comment:
The point of this component is to collapse; I don't think it makes sense to allow disabling.